### PR TITLE
#698 Fix missing import for enums in case of default values.

### LINKED ISSUE #698: @Mapping enum to default doesn't add import for enum
I am trying to map an enum to a default value like this

@Mapping(target = "type", defaultValue = "A")

The target enum is in a different package than the mapper.  When the mapper generates I get a compile error in the generated class saying "cannot find symbol".  I went and looked at the generated code and it all looks correct except there is no import statement for the defaulted enum.

I can get around it if I add a mapping for the enum to itself like below, but I would rather not.

public ObjType toType(ObjType type);

Note, I am using version 1.0.0.CR2 of mapstruct.

### PR DESCRIPTION
**Pull request by agudian**
Fixes #698.

### CODE REVIEW

File: processor/src/test/java/org/mapstruct/ap/test/defaultvalue/CountryMapper.java [hunk 1]
```diff
@@ -40,19 +41,19 @@
             @Mapping( target = "code", defaultValue = "DE" ),
             @Mapping( target = "id", defaultValue = "42" ),
             @Mapping( target = "zipcode", defaultValue = "1337" ),
-            @Mapping( target = "region", ignore = true )
-
+            @Mapping( target = "region", ignore = true ),
+            @Mapping( target = "continent", defaultValue = "EUROPE" )
```
Comment (sjaakd) [inline processor/src/test/java/org/mapstruct/ap/test/defaultvalue/CountryMapper.java]: earth?, solar system? galaxy? local cluster?.. We still have room for more patches :smile:

Comment (gunnarmorling) [inline processor/src/test/java/org/mapstruct/ap/test/defaultvalue/CountryMapper.java]: Ahahaha :)

**Comment by gunnarmorling** [comment]
Merged; Thanks for the quick fix, @agudian!

**Comment by sjaakd** [comment]
I had no more comments either :smile: .. thanks!

**Pull request merged**